src/client/pages/forum/forms/CommentForm.jsx

```diff
@@ -1,12 +1,53 @@
 import { useState} from 'react';
+// import { LOCAL_STORAGE, USER_URL } from '../../../config';
 
 const commentForm = (props) => {
-	const { setShowComment } = props;
+	const { setShowComment, postId, setNewComment, commentId, user } = props;
+	const [comment, setcomment] = useState('');
+	
+	
+	const postComment = async() => {
+		
+		try{
+			const res = await fetch("http://localhost:4000/post/comment", {
```

Comment:
For consistency purposes, I think the URL should be added to the config.js

Also, there is a fetch file in utils.js